### PR TITLE
Mage_Newsletter - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)

### CODE REVIEW

**Comment by sreichel** [comment]
Bump. Easy review.

File: app/code/core/Mage/Newsletter/Helper/Data.php [hunk 1]
```diff
@@ -73,12 +73,11 @@ public function getUnsubscribeUrl($subscriber)
     /**
      * Retrieve Template processor for Newsletter template
      *
-     * @return Varien_Filter_Template
+     * @return false|Mage_Core_Model_Abstract
```
Comment (tmotyl) [inline app/code/core/Mage/Newsletter/Helper/Data.php:76]: Is that right?

File: app/code/core/Mage/Newsletter/Model/Subscriber.php [hunk 2]
```diff
@@ -482,12 +500,12 @@ public function confirm($code)
     /**
      * Mark receiving subscriber of queue newsletter
      *
-     * @param  Mage_Newsletter_Model_Queue $queue
-     * @return boolean
+     * @param Mage_Newsletter_Model_Queue $queue
+     * @return Mage_Newsletter_Model_Subscriber
```
Comment (tmotyl) [inline app/code/core/Mage/Newsletter/Model/Subscriber.php:504]: Why not ‚return $this”?

File: app/code/core/Mage/Newsletter/Model/Template.php [hunk 3]
```diff
@@ -234,6 +238,7 @@ public function getProcessedTemplate(array $variables = array(), $usePreprocess
      * Makes additional text preparations for HTML templates
      *
      * @param bool $usePreprocess Use Preprocessed text or original text
+     * @param null $html
```
Comment (tmotyl) [inline app/code/core/Mage/Newsletter/Model/Template.php:241]: Null/string

Comment (sreichel) [reply app/code/core/Mage/Newsletter/Model/Template.php:241]: Yep. Havent check all methods ... some have just default return type.

**Pull request merged**